### PR TITLE
Re-word bellapps.md and changed the image

### PR DESCRIPTION
**Pull request by Supriyaranjan**

### CODE REVIEW

File: pages/bellapps.md [hunk 1]
```diff
@@ -35,12 +35,13 @@ You will be shown the page below. Make sure you fill it out completely.
 ![Become an Administrator](uploads/images/become_admin.png)
 
 Next, you will need to fill out your configurations. Make sure you write your name all in lowercase letters and your code all in uppercase letters, as in the example below:
```
Comment (waryhermit) [inline pages/bellapps.md]: "... as in the example below:" doesn't make as much sense when there is text after it. Maybe you can rework it to be more clear.

Comment (Supriyaranjan) [inline pages/bellapps.md]: sure

Comment (Supriyaranjan) [inline pages/bellapps.md]: Should I put the "note" like this one?
image

**Comment by Supriyaranjan** [comment]
@waryhermit  @sarvanivadali Is this better?

Next, you will need to fill out your configurations. 
Note that your name and code must be the same and should correspond to your GitHub username.
This is because we should be able to know who owns each community from the nation side.
Also, make sure you pick the correct nation, Virtual Intern Nation (vi), write your name all in lowercase letters and your code all in uppercase letters, as in the example below:

**Comment by Supriyaranjan** [comment]
@waryhermit @sarvanivadali  Is this better?
Next, you will need to fill out your configurations. 
Note that your name and code must be the same and should correspond to your GitHub username.
This is because we should be able to know who owns each community from the nation side.
Also, make sure you write your name all in lowercase letters and your code all in uppercase letters, pick the correct nation, Virtual Intern Nation (vi), as in the example below:

**Comment by waryhermit** [comment]
@Supriyaranjan Hmm better.. How about this:

> Next, fill out the configurations. Your name and code must be the same and should match your Github name. Write your name in lowercase letters and code in uppercase letters, and pick Virtual Intern Nation (vi) for nation as in the example below:

I like short and sweet, and only tell them what they need to know.

File: pages/bellapps.md (hunk 1 above)
Comment (waryhermit) [inline pages/bellapps.md]: @Supriyaranjan No, in fact I would get rid of the word `note`. We seem to use a lot of extra words in the first steps. See my other comment for what I think looks good.

**Pull request merged**